### ISSUE TITLE
FileCreate() returns NO_WACL_KEY

### ISSUE THREAD

**Issue by Cooper-Kunz** [hashgraph/hedera-sdk-js#189]
I'm attempting to create a file without any administrative keys attached. 

Here's the code I want to run.
```
const { Client, FileCreateTransaction, Ed25519PublicKey, Hbar } = require("@hashgraph/sdk");

async function main() {
  const operatorPrivateKey = "...";
  const operatorAccount = "...";

  const client = new Client({
    network: { "0.testnet.hedera.com:50211": "0.0.3" },
    operator: {
      account: operatorAccount,
      privateKey: operatorPrivateKey
    }
  });

  const transactionId = await new FileCreateTransaction()
    .setContents("test file create")
    .setMaxTransactionFee(new Hbar(100))
    .execute(client);

  const receipt = await transactionId.getReceipt(client);  
  console.log("new file id = ", receipt.getFileId());
}

main();
```

Which results in the following error.

```
 HederaStatusError: Hedera returned response code: NO_WACL_KEY
```

Based on the [Hedera protobuf](https://github.com/hashgraph/hedera-protobuf/blob/696830b8e75b68849c6b68eec1e841ef55b5ba1c/src/main/proto/FileCreate.proto#L17) definitions, I should be able to leave this empty.

**Comment by mehcode** [comment]
This has been a bug in the Hedera network. Not much we can do from the SDK side here. Maybe make the error message nicer.

**Comment by Cooper-Kunz** [comment]
Is there a timeline for this getting resolved? I understand I could use a throwaway key in the meantime.

cc @donaldthibeau

**Comment by mehcode** [comment]
It's either a bug in the network or an explicit decision and the proto docs weren't updated. I'm not aware of any movement around it.

**Comment by donaldthibeau** [comment]
This will be fixed for the code to match the proto in the 0.5.0 release.

**Comment by mehcode** [comment]
Closing as this isn't actionable here and it'll get fixed in 0.5 ( nice, :+1: ).
